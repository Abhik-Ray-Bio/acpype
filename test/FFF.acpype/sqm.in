Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   7     N      103.6260       24.3310        5.3960 
   6    CA      105.0710       24.4900        5.2260 
   6     C      105.4270       25.9740        5.0820 
   8     O      104.7730       26.6200        4.2020 
   6    CB      105.5900       23.7580        3.9530 
   6    CG      105.3690       22.2380        3.8340 
   6   CD1      104.2650       21.7910        3.0870 
   6   CD2      106.2130       21.3040        4.4300 
   6   CE1      103.9740       20.4360        3.0170 
   6   CE2      105.9410       19.9420        4.3250 
   6    CZ      104.8050       19.5090        3.6420 
   1    HA      105.5730       24.0900        6.1220 
   1   HHB      106.6780       23.9450        3.8520 
   1    H1      105.1430       24.2370        3.0550 
   1   HHD      103.6450       22.5150        2.5700 
   1    H2      107.0870       21.6330        4.9780 
   1   HHE      103.0960       20.1110        2.4740 
   1    H3      106.6080       19.2170        4.7700 
   1    HZ      104.5730       18.4550        3.6000 
   1   HHT      103.3920       23.3360        5.5050 
   1    H4      103.3220       24.8470        6.2320 
   1    H5      103.1440       24.7080        4.5650 
   7     N      106.4270       26.4990        5.9860 
   6    CA      106.5910       27.9400        5.9910 
   6     C      108.0820       28.2940        6.1380 
   8     O      108.6900       27.7650        7.1210 
   6    CB      105.8400       28.5860        7.1990 
   6    CG      104.3110       28.3940        7.3070 
   6   CD1      103.8560       27.3950        8.1880 
   6   CD2      103.4040       29.1280        6.5520 
   6   CE1      102.5050       27.0830        8.2280 
   6   CE2      102.0450       28.8220        6.6170 
   6    CZ      101.6020       27.7840        7.4350 
   1     H      107.0030       25.8520        6.6780 
   1    HA      106.2600       28.4240        5.0650 
   1   HHB      106.0410       29.6750        7.2050 
   1    H6      106.2990       28.2200        8.1420 
   1   HHD      104.5830       26.8850        8.8140 
   1    H7      103.7590       29.9300        5.9140 
   1   HHE      102.1640       26.2870        8.8780 
   1    H8      101.3330       29.3870        6.0320 
   1    HZ      100.5540       27.5220        7.4530 
   7     N      108.6460       29.1600        5.1260 
   6    CA      110.0820       29.3620        5.1800 
   6     C      110.4300       30.8050        4.8490 
   8     O      109.6750       31.6290        4.2260 
   6    CB      110.8070       28.4480        4.1350 
   6    CG      110.6430       26.9160        4.2480 
   6   CD1      109.6940       26.3070        3.4060 
   6   CD2      111.3590       26.1530        5.1650 
   6   CE1      109.4080       24.9590        3.5650 
   6   CE2      111.0790       24.7950        5.2980 
   6    CZ      110.0860       24.2060        4.5190 
   8   OXT      111.5900       31.1670        5.2390 
   1     H      108.0300       29.6260        4.3290 
   1    HA      110.5170       29.1850        6.1730 
   1   HHB      111.8920       28.6660        4.1660 
   1    H9      110.4920       28.7530        3.1160 
   1   HHD      109.2010       26.9150        2.6540 
   1   H10      112.1250       26.6240        5.7710 
   1   HHE      108.6490       24.5000        2.9420 
   1   H11      111.6290       24.1940        6.0100 
   1    HZ      109.8410       23.1620        4.6560 

