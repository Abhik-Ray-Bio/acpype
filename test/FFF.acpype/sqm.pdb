ATOM      1  N   PHE     1     106.711  24.493   3.738  1.00  0.00           N
ATOM      2  CA  PHE     1     105.394  24.470   4.450  1.00  0.00           C
ATOM      3  C   PHE     1     104.978  25.955   4.663  1.00  0.00           C
ATOM      4  O   PHE     1     104.684  26.586   3.636  1.00  0.00           O
ATOM      5  CB  PHE     1     105.473  23.626   5.722  1.00  0.00           C
ATOM      6  CG  PHE     1     105.810  22.213   5.396  1.00  0.00           C
ATOM      7  CD1 PHE     1     107.130  21.761   5.518  1.00  0.00           C
ATOM      8  CD2 PHE     1     104.810  21.324   4.988  1.00  0.00           C
ATOM      9  CE1 PHE     1     107.444  20.434   5.228  1.00  0.00           C
ATOM     10  CE2 PHE     1     105.128  19.998   4.698  1.00  0.00           C
ATOM     11  CZ  PHE     1     106.445  19.552   4.816  1.00  0.00           C
ATOM     12  HA  PHE     1     104.637  24.007   3.747  1.00  0.00           H
ATOM     13  HHB PHE     1     104.471  23.675   6.229  1.00  0.00           H
ATOM     14  H1  PHE     1     106.245  24.052   6.425  1.00  0.00           H
ATOM     15  HHD PHE     1     107.917  22.446   5.871  1.00  0.00           H
ATOM     16  H2  PHE     1     103.767  21.669   4.908  1.00  0.00           H
ATOM     17  HHE PHE     1     108.482  20.081   5.335  1.00  0.00           H
ATOM     18  H3  PHE     1     104.336  19.301   4.382  1.00  0.00           H
ATOM     19  HZ  PHE     1     106.694  18.503   4.591  1.00  0.00           H
ATOM     20  HHT PHE     1     107.028  23.537   3.553  1.00  0.00           H
ATOM     21  H4  PHE     1     106.628  25.008   2.850  1.00  0.00           H
ATOM     22  H5  PHE     1     107.433  24.970   4.320  1.00  0.00           H
ATOM     23  N   PHE     2     104.957  26.512   5.941  1.00  0.00           N
ATOM     24  CA  PHE     2     105.787  27.706   6.102  1.00  0.00           C
ATOM     25  C   PHE     2     107.273  27.366   5.772  1.00  0.00           C
ATOM     26  O   PHE     2     107.641  26.153   5.860  1.00  0.00           O
ATOM     27  CB  PHE     2     105.738  28.282   7.523  1.00  0.00           C
ATOM     28  CG  PHE     2     104.369  28.712   7.910  1.00  0.00           C
ATOM     29  CD1 PHE     2     103.657  28.012   8.891  1.00  0.00           C
ATOM     30  CD2 PHE     2     103.790  29.839   7.313  1.00  0.00           C
ATOM     31  CE1 PHE     2     102.381  28.432   9.266  1.00  0.00           C
ATOM     32  CE2 PHE     2     102.514  30.255   7.689  1.00  0.00           C
ATOM     33  CZ  PHE     2     101.807  29.551   8.664  1.00  0.00           C
ATOM     34  H   PHE     2     105.105  25.858   6.686  1.00  0.00           H
ATOM     35  HA  PHE     2     105.397  28.478   5.372  1.00  0.00           H
ATOM     36  HHB PHE     2     106.432  29.169   7.549  1.00  0.00           H
ATOM     37  H6  PHE     2     106.135  27.526   8.252  1.00  0.00           H
ATOM     38  HHD PHE     2     104.108  27.134   9.376  1.00  0.00           H
ATOM     39  H7  PHE     2     104.350  30.401   6.551  1.00  0.00           H
ATOM     40  HHE PHE     2     101.829  27.880  10.040  1.00  0.00           H
ATOM     41  H8  PHE     2     102.067  31.142   7.217  1.00  0.00           H
ATOM     42  HZ  PHE     2     100.801  29.882   8.962  1.00  0.00           H
ATOM     43  N   PHE     3     108.072  28.349   5.316  1.00  0.00           N
ATOM     44  CA  PHE     3     109.498  28.186   5.103  1.00  0.00           C
ATOM     45  C   PHE     3     110.304  29.431   5.693  1.00  0.00           C
ATOM     46  O   PHE     3     109.628  30.422   6.069  1.00  0.00           O
ATOM     47  CB  PHE     3     109.884  28.072   3.625  1.00  0.00           C
ATOM     48  CG  PHE     3     109.465  26.804   2.974  1.00  0.00           C
ATOM     49  CD1 PHE     3     108.606  26.817   1.866  1.00  0.00           C
ATOM     50  CD2 PHE     3     109.960  25.572   3.430  1.00  0.00           C
ATOM     51  CE1 PHE     3     108.257  25.629   1.223  1.00  0.00           C
ATOM     52  CE2 PHE     3     109.605  24.386   2.791  1.00  0.00           C
ATOM     53  CZ  PHE     3     108.756  24.410   1.682  1.00  0.00           C
ATOM     54  OXT PHE     3     111.546  29.284   5.717  1.00  0.00           O
ATOM     55  H   PHE     3     107.800  29.309   5.449  1.00  0.00           H
ATOM     56  HA  PHE     3     109.850  27.266   5.650  1.00  0.00           H
ATOM     57  HHB PHE     3     111.010  28.145   3.600  1.00  0.00           H
ATOM     58  H9  PHE     3     109.480  28.953   3.058  1.00  0.00           H
ATOM     59  HHD PHE     3     108.221  27.777   1.488  1.00  0.00           H
ATOM     60  H10 PHE     3     110.646  25.556   4.294  1.00  0.00           H
ATOM     61  HHE PHE     3     107.614  25.660   0.330  1.00  0.00           H
ATOM     62  H11 PHE     3     110.030  23.431   3.136  1.00  0.00           H
ATOM     63  HZ  PHE     3     108.524  23.480   1.144  1.00  0.00           H
