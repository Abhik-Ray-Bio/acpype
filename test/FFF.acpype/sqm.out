            --------------------------------------------------------
                             AMBER SQM VERSION 17
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 88

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =      165
| QMMM:    diag iterations used for timing =        5
| QMMM:
| QMMM:              Internal diag routine = 0.071698 seconds
| QMMM:                 Dspev diag routine = 0.091930 seconds
| QMMM:                Dspevd diag routine = 0.064216 seconds
| QMMM:                Dspevx diag routine = 0.206939 seconds
| QMMM:                 Dsyev diag routine = 0.098292 seconds
| QMMM:                Dsyevd diag routine = 0.070726 seconds
| QMMM:                Dsyevr diag routine = 0.063427 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.033437 seconds
| QMMM:
| QMMM: Using dsyevr routine (diag_routine=7).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      N      103.6260   24.3310    5.3960
  QMMM:     2        2      C      105.0710   24.4900    5.2260
  QMMM:     3        3      C      105.4270   25.9740    5.0820
  QMMM:     4        4      O      104.7730   26.6200    4.2020
  QMMM:     5        5      C      105.5900   23.7580    3.9530
  QMMM:     6        6      C      105.3690   22.2380    3.8340
  QMMM:     7        7      C      104.2650   21.7910    3.0870
  QMMM:     8        8      C      106.2130   21.3040    4.4300
  QMMM:     9        9      C      103.9740   20.4360    3.0170
  QMMM:    10       10      C      105.9410   19.9420    4.3250
  QMMM:    11       11      C      104.8050   19.5090    3.6420
  QMMM:    12       12      H      105.5730   24.0900    6.1220
  QMMM:    13       13      H      106.6780   23.9450    3.8520
  QMMM:    14       14      H      105.1430   24.2370    3.0550
  QMMM:    15       15      H      103.6450   22.5150    2.5700
  QMMM:    16       16      H      107.0870   21.6330    4.9780
  QMMM:    17       17      H      103.0960   20.1110    2.4740
  QMMM:    18       18      H      106.6080   19.2170    4.7700
  QMMM:    19       19      H      104.5730   18.4550    3.6000
  QMMM:    20       20      H      103.3920   23.3360    5.5050
  QMMM:    21       21      H      103.3220   24.8470    6.2320
  QMMM:    22       22      H      103.1440   24.7080    4.5650
  QMMM:    23       23      N      106.4270   26.4990    5.9860
  QMMM:    24       24      C      106.5910   27.9400    5.9910
  QMMM:    25       25      C      108.0820   28.2940    6.1380
  QMMM:    26       26      O      108.6900   27.7650    7.1210
  QMMM:    27       27      C      105.8400   28.5860    7.1990
  QMMM:    28       28      C      104.3110   28.3940    7.3070
  QMMM:    29       29      C      103.8560   27.3950    8.1880
  QMMM:    30       30      C      103.4040   29.1280    6.5520
  QMMM:    31       31      C      102.5050   27.0830    8.2280
  QMMM:    32       32      C      102.0450   28.8220    6.6170
  QMMM:    33       33      C      101.6020   27.7840    7.4350
  QMMM:    34       34      H      107.0030   25.8520    6.6780
  QMMM:    35       35      H      106.2600   28.4240    5.0650
  QMMM:    36       36      H      106.0410   29.6750    7.2050
  QMMM:    37       37      H      106.2990   28.2200    8.1420
  QMMM:    38       38      H      104.5830   26.8850    8.8140
  QMMM:    39       39      H      103.7590   29.9300    5.9140
  QMMM:    40       40      H      102.1640   26.2870    8.8780
  QMMM:    41       41      H      101.3330   29.3870    6.0320
  QMMM:    42       42      H      100.5540   27.5220    7.4530
  QMMM:    43       43      N      108.6460   29.1600    5.1260
  QMMM:    44       44      C      110.0820   29.3620    5.1800
  QMMM:    45       45      C      110.4300   30.8050    4.8490
  QMMM:    46       46      O      109.6750   31.6290    4.2260
  QMMM:    47       47      C      110.8070   28.4480    4.1350
  QMMM:    48       48      C      110.6430   26.9160    4.2480
  QMMM:    49       49      C      109.6940   26.3070    3.4060
  QMMM:    50       50      C      111.3590   26.1530    5.1650
  QMMM:    51       51      C      109.4080   24.9590    3.5650
  QMMM:    52       52      C      111.0790   24.7950    5.2980
  QMMM:    53       53      C      110.0860   24.2060    4.5190
  QMMM:    54       54      O      111.5900   31.1670    5.2390
  QMMM:    55       55      H      108.0300   29.6260    4.3290
  QMMM:    56       56      H      110.5170   29.1850    6.1730
  QMMM:    57       57      H      111.8920   28.6660    4.1660
  QMMM:    58       58      H      110.4920   28.7530    3.1160
  QMMM:    59       59      H      109.2010   26.9150    2.6540
  QMMM:    60       60      H      112.1250   26.6240    5.7710
  QMMM:    61       61      H      108.6490   24.5000    2.9420
  QMMM:    62       62      H      111.6290   24.1940    6.0100
  QMMM:    63       63      H      109.8410   23.1620    4.6560

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10      -23.9060 kcal/mol        1.3358 kcal/(mol*A)
xmin    20      -25.3563 kcal/mol        0.2796 kcal/(mol*A)
xmin    30      -26.0936 kcal/mol        0.4528 kcal/(mol*A)
xmin    40      -26.6087 kcal/mol        0.1831 kcal/(mol*A)
xmin    50      -27.0087 kcal/mol        0.2569 kcal/(mol*A)
xmin    60      -28.0258 kcal/mol        0.5087 kcal/(mol*A)
xmin    70      -28.5311 kcal/mol        0.0911 kcal/(mol*A)
xmin    80      -29.0602 kcal/mol        0.6285 kcal/(mol*A)
xmin    90      -36.4218 kcal/mol        0.9558 kcal/(mol*A)
xmin   100      -39.5731 kcal/mol        0.6924 kcal/(mol*A)
xmin   110      -41.9594 kcal/mol        0.1555 kcal/(mol*A)
xmin   120      -42.1373 kcal/mol        0.0578 kcal/(mol*A)
xmin   130      -42.1602 kcal/mol        0.0721 kcal/(mol*A)
xmin   140      -42.1814 kcal/mol        0.0201 kcal/(mol*A)
xmin   150      -42.1851 kcal/mol        0.0276 kcal/(mol*A)
xmin   160      -42.1885 kcal/mol        0.0109 kcal/(mol*A)
xmin   170      -42.1892 kcal/mol        0.0027 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -42.8265  -41.0688  -40.4663  -40.2692  -39.5270  -39.0132  -37.4147  -36.5519
 -35.2533  -34.9370  -34.0757  -33.6335  -32.8274  -32.5293  -32.1930  -31.6860
 -31.2982  -30.1332  -28.9677  -27.1918  -25.9873  -24.6762  -24.5701  -23.9214
 -23.6048  -23.4659  -22.8718  -22.2754  -21.8704  -21.2779  -21.0764  -20.7407
 -19.6662  -19.3026  -18.8552  -18.5576  -18.2117  -18.1149  -17.7783  -17.7066
 -17.2779  -17.1197  -16.8224  -16.7271  -16.5961  -16.3384  -16.0535  -16.0097
 -15.9015  -15.6709  -15.5350  -15.3715  -15.1924  -14.9921  -14.9085  -14.6899
 -14.5564  -14.4527  -14.4144  -14.2892  -14.2826  -13.9960  -13.6183  -13.5052
 -13.2566  -13.2324  -13.1703  -13.0591  -12.9976  -12.7881  -12.7070  -12.3989
 -12.3179  -12.2344  -12.0567  -12.0109  -11.5260  -11.0743  -11.0000  -10.8336
 -10.5582  -10.4071   -9.9282   -9.9062   -9.7236   -8.3392   -7.9780   -7.5911
  -1.2387   -1.1012   -0.8451   -0.4246   -0.3443   -0.2404    0.1501    0.3172
   0.3646    0.7346    1.0983    1.2948    1.4366    1.5948    1.7955    1.8150
   2.0076    2.0583    2.2054    2.3348    2.3806    2.5913    2.6383    2.7772
   2.9114    2.9158    3.0141    3.1442    3.1744    3.2179    3.2639    3.2754
   3.2851    3.3274    3.5492    3.6724    3.6772    3.6982    3.7575    3.7664
   3.7762    3.8293    3.8509    3.9252    3.9841    4.1382    4.1989    4.2320
   4.2522    4.3151    4.3334    4.3555    4.4119    4.5515    4.6428    4.6696
   4.7028    4.7461    4.8487    4.8713    4.8829    4.9535    5.1092    5.1332
   5.1753    5.3199    5.3356    5.3495    5.3921    5.4692    5.6526    5.6824
   5.8562    6.4783    6.7128    9.1225   10.3504

 Heat of formation   =        -42.18917016 kcal/mol  (       -1.82945970 eV)

 Total SCF energy    =    -132753.04095209 kcal/mol  (    -5756.60383123 eV)
 Electronic energy   =   -1205465.64829379 kcal/mol  (   -52272.91306942 eV)
 Core-core repulsion =    1072712.60734169 kcal/mol  (    46516.30923818 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      N                 -0.075
     2      C                 -0.108
     3      C                  0.274
     4      O                 -0.296
     5      C                 -0.150
     6      C                 -0.131
     7      C                 -0.141
     8      C                 -0.120
     9      C                 -0.121
    10      C                 -0.121
    11      C                 -0.111
    12      H                  0.152
    13      H                  0.132
    14      H                  0.142
    15      H                  0.145
    16      H                  0.138
    17      H                  0.148
    18      H                  0.146
    19      H                  0.147
    20      H                  0.249
    21      H                  0.265
    22      H                  0.299
    23      N                 -0.275
    24      C                 -0.022
    25      C                  0.249
    26      O                 -0.522
    27      C                 -0.126
    28      C                 -0.077
    29      C                 -0.123
    30      C                 -0.118
    31      C                 -0.129
    32      C                 -0.123
    33      C                 -0.125
    34      H                  0.208
    35      H                  0.148
    36      H                  0.135
    37      H                  0.104
    38      H                  0.126
    39      H                  0.136
    40      H                  0.131
    41      H                  0.136
    42      H                  0.133
    43      N                 -0.263
    44      C                 -0.115
    45      C                  0.346
    46      O                 -0.548
    47      C                 -0.140
    48      C                 -0.031
    49      C                 -0.142
    50      C                 -0.126
    51      C                 -0.165
    52      C                 -0.170
    53      C                 -0.210
    54      O                 -0.511
    55      H                  0.265
    56      H                  0.111
    57      H                  0.150
    58      H                  0.101
    59      H                  0.151
    60      H                  0.164
    61      H                  0.138
    62      H                  0.135
    63      H                  0.131
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE   -13.870  -18.385   -5.044   23.576
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      N      106.7110   24.4934    3.7377
  QMMM:     2        2      C      105.3939   24.4702    4.4502
  QMMM:     3        3      C      104.9777   25.9553    4.6632
  QMMM:     4        4      O      104.6842   26.5864    3.6356
  QMMM:     5        5      C      105.4733   23.6263    5.7222
  QMMM:     6        6      C      105.8103   22.2133    5.3959
  QMMM:     7        7      C      107.1301   21.7611    5.5176
  QMMM:     8        8      C      104.8098   21.3238    4.9878
  QMMM:     9        9      C      107.4443   20.4341    5.2278
  QMMM:    10       10      C      105.1279   19.9979    4.6976
  QMMM:    11       11      C      106.4448   19.5521    4.8155
  QMMM:    12       12      H      104.6366   24.0070    3.7468
  QMMM:    13       13      H      104.4714   23.6750    6.2292
  QMMM:    14       14      H      106.2448   24.0521    6.4253
  QMMM:    15       15      H      107.9172   22.4463    5.8712
  QMMM:    16       16      H      103.7675   21.6687    4.9076
  QMMM:    17       17      H      108.4817   20.0811    5.3346
  QMMM:    18       18      H      104.3364   19.3010    4.3823
  QMMM:    19       19      H      106.6937   18.5035    4.5915
  QMMM:    20       20      H      107.0284   23.5370    3.5533
  QMMM:    21       21      H      106.6277   25.0082    2.8497
  QMMM:    22       22      H      107.4328   24.9701    4.3202
  QMMM:    23       23      N      104.9575   26.5123    5.9413
  QMMM:    24       24      C      105.7873   27.7062    6.1016
  QMMM:    25       25      C      107.2732   27.3658    5.7724
  QMMM:    26       26      O      107.6408   26.1531    5.8600
  QMMM:    27       27      C      105.7380   28.2816    7.5229
  QMMM:    28       28      C      104.3686   28.7117    7.9102
  QMMM:    29       29      C      103.6566   28.0124    8.8908
  QMMM:    30       30      C      103.7899   29.8388    7.3133
  QMMM:    31       31      C      102.3807   28.4317    9.2656
  QMMM:    32       32      C      102.5141   30.2545    7.6886
  QMMM:    33       33      C      101.8070   29.5513    8.6643
  QMMM:    34       34      H      105.1050   25.8577    6.6856
  QMMM:    35       35      H      105.3975   28.4783    5.3725
  QMMM:    36       36      H      106.4317   29.1685    7.5492
  QMMM:    37       37      H      106.1354   27.5260    8.2524
  QMMM:    38       38      H      104.1076   27.1340    9.3755
  QMMM:    39       39      H      104.3495   30.4015    6.5507
  QMMM:    40       40      H      101.8289   27.8797   10.0405
  QMMM:    41       41      H      102.0668   31.1421    7.2169
  QMMM:    42       42      H      100.8013   29.8824    8.9617
  QMMM:    43       43      N      108.0717   28.3486    5.3155
  QMMM:    44       44      C      109.4983   28.1859    5.1026
  QMMM:    45       45      C      110.3038   29.4306    5.6926
  QMMM:    46       46      O      109.6280   30.4219    6.0688
  QMMM:    47       47      C      109.8839   28.0723    3.6246
  QMMM:    48       48      C      109.4652   26.8037    2.9735
  QMMM:    49       49      C      108.6055   26.8169    1.8663
  QMMM:    50       50      C      109.9598   25.5717    3.4302
  QMMM:    51       51      C      108.2567   25.6294    1.2227
  QMMM:    52       52      C      109.6052   24.3857    2.7913
  QMMM:    53       53      C      108.7557   24.4096    1.6823
  QMMM:    54       54      O      111.5459   29.2842    5.7166
  QMMM:    55       55      H      107.8002   29.3092    5.4489
  QMMM:    56       56      H      109.8505   27.2660    5.6496
  QMMM:    57       57      H      111.0105   28.1448    3.5999
  QMMM:    58       58      H      109.4803   28.9526    3.0576
  QMMM:    59       59      H      108.2205   27.7775    1.4881
  QMMM:    60       60      H      110.6457   25.5559    4.2936
  QMMM:    61       61      H      107.6142   25.6601    0.3296
  QMMM:    62       62      H      110.0304   23.4310    3.1360
  QMMM:    63       63      H      108.5240   23.4796    1.1437

           --------- Calculation Completed ----------

